src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastHelpers.cs

```diff
@@ -316,6 +319,10 @@ internal static unsafe class CastHelpers
             return ChkCastClassSpecial(toTypeHnd, obj);
         }
 
+        [DebuggerHidden]
+        private static int ChkCastTypeToType(void* fromTypeHnd, void* toTypeHnd)
```

Comment:
I'm just going with what the JIT is expecting. In `importer.cpp`, I am inserting code with a return type of `TYP_INT` so am following that pattern. Is returning managed `bool` here appropriate?

Reply:
`IsAssignable` makes sense too. I will take that suggestion.

Reply:
> Is returning managed bool here appropriate?

It would look better. (There should not be any low-level difference since bool is physically returned as INT32 in managed calling convention.)

Reply:
Okay.